src/mmw/js/src/core/modals/views.js

```diff
@@ -296,6 +298,56 @@ var MultiShareView = ModalBaseView.extend({
     }
 });
 
+var HydroShareView = ModalBaseView.extend({
+    className: LARGE_MODAL_CLASS,
+    template: modalHydroShareTmpl,
+
+    ui: {
+        'title': '#hydroshare-title',
+        'abstract': '#hydroshare-abstract',
+        'keywords': '#hydroshare-keywords',
+        'export': '.btn-active',
+        'cancel': '.btn-default',
+    },
+
+    events: _.defaults({
+        'click @ui.export': 'primaryAction',
+        'click @ui.cancel': 'dismissAction',
+    }, ModalBaseView.prototype.events),
+
+    primaryAction: function() {
+        var title = this.ui.title.val().trim(),
+            abstract = this.ui.abstract.val().trim(),
+            keywords = this.ui.keywords.val().trim();
+
+        if (title === "" || abstract === "") {
+            return;
```

Comment:
Since the user does not receive any feedback letting them know that these fields are required the export button appears to do nothing if both fields are not filled in. A static "required" label near fields may be all we need.

Reply:
Added a small `(optional)` text to keywords which hopefully makes the other two more clearly required:

![image](https://user-images.githubusercontent.com/1430060/34419457-08fbb7c4-ebd2-11e7-9c46-10b29aaf19f3.png)